pkg/vsphere/disk/vmdk.go

```diff
@@ -58,7 +64,34 @@ func LockedVMDKFilter(vm *mo.VirtualMachine) bool {
 	return vm.Runtime.PowerState == types.VirtualMachinePowerStatePoweredOn
 }
 
-func IsLockedError(err error) bool {
-	// TODO: add check here for actual error
-	return true
+func IsLockedError(op trace.Operation, err error) bool {
```

Comment:
okay, anyway, didin't figure out many difference between these two methods. But the method LockedDisks will do more than isLockedError, so we'll have isLockedError call LockedDisks, just do not return the locked disks.

So we'll have two methods for different users with different interest, but the logic will be in LockedDisks.